Makefile

```diff
@@ -114,13 +114,9 @@ version:
 vendor:
 	$(GO) mod vendor
 
-.PHONY: oapi-gen-types
-oapi-gen-types: ;
-	oapi-codegen --generate types -o driver/web/docs/gen/gen_types.go driver/web/docs/gen/def.yaml
-
-.PHONY: oapi-gen-docs
-oapi-gen-docs: ;
-	swagger-cli bundle driver/web/docs/index.yaml --outfile driver/web/docs/gen/def.yaml --type yaml	
+.PHONY: swagger
+swagger: ;
+	swag init -g driver/web/adapter.go
```

Comment:
This change was intentional as we are using the manually defined documentation strategy used on the Core BB on this BB now.

Reply:
Thanks I was unaware for that. I will redo this part.